tests/blockchain/test_blockchain.py

```diff
@@ -2055,6 +2108,8 @@ async def test_ephemeral_timelock(self, opcode, lock_value, expected, with_garba
                 ConditionOpcode.ASSERT_MY_BIRTH_SECONDS,
                 ConditionOpcode.ASSERT_SECONDS_RELATIVE,
                 ConditionOpcode.ASSERT_HEIGHT_RELATIVE,
+                ConditionOpcode.ASSERT_BEFORE_HEIGHT_RELATIVE,
+                ConditionOpcode.ASSERT_BEFORE_SECONDS_RELATIVE,
```

Comment:
these are now added as special cases for both before- and after the softfork. make the test cases in the list above reflect one or the other